src/util/QdmAttributeHelpers.ts

```diff
@@ -86,6 +87,67 @@ export const determineAttributeTypeList = (path, info) => {
   else return [info.instance];
 };
 
+// This is specific to DataElements Table as multiple data types from same attribute has to be displayed in same cell
+export const generateAttributesToDisplay = (
+  dataElement,
+  dataElements,
+  codeSystemMap: any
+) => {
+  const dataElementClass = getDataElementClass(dataElement);
+  const modeledEl = new dataElementClass(dataElement);
```

Comment:
this is just creating a new instance of data element. is there anything that is not available in `dataElement` that forces you to create a new instance?

Reply:
Yes, we need Schema, thats not available in our JSON